specs/order_spec.rb

```diff
@@ -141,21 +180,79 @@
     end
 
     it "Returns accurate information about the last order" do
-      # TODO: Your test code here!
+      id = 100
+      products = {
+        "Amaranth" => 83.81,
+        "Smoked Trout" => 70.6,
+        "Cheddar" => 5.63
+      }
+      customer_id = 20
+      fulfillment_status = :pending
```

Comment:
after running `rake`, Minitest will complain and giving a warning that these variables above (`id`, `products`, etc) are being declared, but not being used, so you should just delete them. I think that's safe to do!